demos/slider.html

```diff
@@ -109,56 +109,122 @@
+
+          <div class="example-slider-wrapper">
+            <div id="discrete-mdc-slider-w-marker" class="mdc-slider mdc-slider--discrete mdc-slider--display-markers" tabindex="0" role="slider"
+                 aria-valuemin="0" aria-valuemax="100" aria-valuenow="30"
+                 aria-labelledby="example-ds-marker-label">
+              <div class="mdc-slider__track-container">
+                <div class="mdc-slider__track"></div>
+                <div class="mdc-slider__track-marker-container">
+                </div>
+              </div>
+              <div class="mdc-slider__thumb-container">
+                <div class="mdc-slider__pin">
+                  <span class="mdc-slider__pin-value-marker">30</span>
+                </div>
+                <svg class="mdc-slider__thumb" width="21" height="21">
+                  <circle cx="10.5" cy="10.5" r="7.875"></circle>
+                </svg>
+                <div class="mdc-slider__focus-ring"></div>
+              </div>
+            </div>
           </div>
+
           <p>
-            Value from <code>MDCSlider:input</code> event: <span class="value">0</span>
+            Value from <code>MDCSlider:input</code> event: <span id="discrete-slider-w-marker-value">0</span>
           </p>
           <p>
-            Value from <code>MDCSlider:change</code> event: <span class="committed-value">0</span>
+            Value from <code>MDCSlider:change</code> event: <span id="discrete-slider-w-marker-committed-value">0</span>
           </p>
         </div>
+
+        <div>
+          <label>
+            Min: <input name="min" type="number" min="0" max="100" value="0">
+          </label>
+        </div>
+        <div>
+          <label>
+            Max: <input name="max" type="number" min="0" max="100" value="100">
+          </label>
+        </div>
+        <div>
+          <label>
+            Step: <input name="step" type="number" min="0" max="100" value="0">
+          </label>
+        </div>
+        <div>
+          <label>
+            Dark Theme: <input type="checkbox" name="dark-theme">
+          </label>
+        </div>
+        <div>
+          <label>
+            Disabled: <input type="checkbox" name="disabled">
+          </label>
+        </div>
+        <div>
+          <label>
+            Use Custom BG Color: <input type="checkbox" name="use-custom-color">
+          </label>
+        </div>
+        <div>
+          <label>
+            RTL: <input type="checkbox" name="rtl">
+          </label>
+        </div>
       </section>
     </main>
     <script src="assets/material-components-web.js" charset="utf-8"></script>
     <script>
       (function() {
         setTimeout(function() {
           mdc.slider.MDCSlider.attachTo(document.getElementById('hero-slider'));
-          initDemo(document.getElementById('continuous-slider-example'));
+          initDemo(document.getElementById('slider-example'));
         }, 80);
```

Comment:
We need to keep this one as the style loader we use sometime will not be fully loaded when the page runs to the end of this script. It will unstably have some funny weird effect if they are not initialize correctly. We have similar setup in `toolbar` and `dialog`.